styleguide/yaml-metadata.md

```diff
@@ -0,0 +1,104 @@
+# YAML metadata in Tizen documentents
+
+This guideline describes YAML syntax and YAML meta.
+
+The following code shows the YAML metadata in Tizen documents.
+
+## Syntax
+
+All YAML files can optionally begin with `---` and end with `---`. This is part of the YAML format and indicates the start and end of a document.
+
+
+A dictionary is represented in a simple `key: value` form (the colon must be followed by a space):
+```
+---
+title: YAML metadata in Tizen documentents
+author: Anonymous (anonymous@tizen.org)
+summary: YAML metadata guideline
+keyword: YAML, metadata, Tizen, YAML syntax
+redirect: overview
+---
+```
+
+## YAML elements
+
+The Tizen documentation has various YAML elements.
+
+### title 
+
+There is no title in menu, the title in meta displays displays this title.
+
+```
+---
+title: YAML metadata in Tizen documentents
+---
+```
+
+### author
+
+The Tizen repository is in public GitHub so we don't know the exact the document authors.
+When the authors are listed, it helps to contact the authors when there are issues.
```

Comment:
Small question. If another person, who is not the author updates or enhance the page, how should we change the author?